### PR TITLE
[Feature] Add queries 28 b and c

### PR DESCRIPTION
**Pull request by esirK**
## Description

Fix sr05 to show all media types and generate reports for regions and countries.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### ws_28b
Screenshot 2021-04-07 at 16 51 51

### ws_28c
Screenshot 2021-04-08 at 09 34 23

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Review by kilemensi — COMMENTED** [review]
👍🏼 @esirK 

———-

1. `ws_28b` _reads_ alright to me, let’s add a screenshot
2. Let’s confirm `ws_sr05` changes; I’d rather we implement `ws_28c` than change any of the `sr` this late.

**Comment by esirK** [comment]
@kilemensi  I think It's ready now.

File: reports/report_builder.py [hunk 1]
```diff
@@ -108,14 +108,16 @@ def sheet_name_to_num(sheet):
     Sheets are banded: 0 - 199 normal sheets, 200 - 399 s sheets, and 400+ for
     sr sheets.
     """
-    stripped_sheet = sheet.strip("wsrb_")
+    stripped_sheet = sheet.strip("wsrb_").strip("wsrc_")
```
Comment (kilemensi) [inline reports/report_builder.py]:
```suggestion
    stripped_sheet = sheet.strip("wsrbc_")
```

File: reports/report_builder.py [hunk 2]
```diff
@@ -1228,6 +1230,154 @@ def ws_28(self, ws):
...
+                        counts.update({(row['sex'], region_id): row['n']})
+
+                    secondary_counts[journo_type] = counts
+
+                self.tabulate_secondary_cols(ws, secondary_counts, self.male_female, all_regions, row_perc=True, show_N=True, c=c, r=r, write_row_headings=write_row_headings)
+
+                c += (len(reporter) * len(self.male_female) * 2) + (1 if write_row_headings else 0)
+                write_row_headings = False
+    
+    def ws_28c(self, ws):
```
Comment (kilemensi) [inline reports/report_builder.py:1310]:
```suggestion
    def ws_28c(self, ws):
        """
        Cols: Media; Journo Type; Sex
        Rows: Country
        :: Radio, Television by region and country
        """
```

File: reports/report_builder.py [hunk 3]
```diff
@@ -1228,6 +1230,154 @@ def ws_28(self, ws):
             self.write_overall_value(ws, value, total, overall_column, overall_row, write_overall=False)
             overall_column +=4
 
+    def ws_28b(self, ws):
+        """
+        Cols: Media; Journo Type; Sex
+        Rows: Country
+        :: Newspaper, Television, Radio, Twitter, Internet by global, region and country
```
Comment (kilemensi) [inline reports/report_builder.py]:
```suggestion
        :: Newspaper, Television, Radio, Twitter, Internet by region and country
```

Do we have by "global"?

**Review by kilemensi — COMMENTED** [review]
👍🏾 @esirK 

------------------

Can you may be explain bit: 
   i. Why are we adding Transnational to `28b` and `28c` but we don't have it in `28`?
   ii. For `28c`, I thought it was supposed to be _Presenters_  only i.e. in Radio and Television only but I see we even have Print, etc.

File: reports/report_builder.py (hunk 3 above)
Comment (esirK) [reply reports/report_builder.py]: >Do we have by "global"?

Currently, we are generating for all 3. Should I remove the global one and only have region and country reports for this one?

**Comment by esirK** [comment]
Can you may be explain bit:
> i. Why are we adding Transnational to `28b` and `28c` but we don't have it in `28`?

This is because they are supposed to be exactly as `ws_sr05`(which has Transnational) the only difference being they are also displayed for countries and regions.
> ii. For `28c`, I thought it was supposed to be _Presenters_ only i.e. in Radio and Television only but I see we even have Print, etc.

Yes, that is true. I've fixed that up.

@kilemensi

File: reports/report_builder.py (hunk 3 above)
Comment (kilemensi) [reply reports/report_builder.py]: No! I meant the **output format** not _report type_. Report type should remain all 3 but output format is only region and country. My understanding was that comment refers to the output format i.e. column, rows, and any other information.

Comment (esirK) [reply reports/report_builder.py]: Ohh. I see now.

**Comment by kilemensi** [comment]
> Can you may be explain bit:
> 
> > i. Why are we adding Transnational to `28b` and `28c` but we don't have it in `28`?
> 
> This is because they are supposed to be exactly as `ws_sr05`(which has Transnational) the only difference being they are also displayed for countries and regions.
> 
> > ii. For `28c`, I thought it was supposed to be _Presenters_ only i.e. in Radio and Television only but I see we even have Print, etc.
> 
> Yes, that is true. I've fixed that up.
> 
> @kilemensi

Cool, cool @esirK . So no need for Transnational in `28` itself? I'm assuming `28`, `28b` and `28c` are somewhat related and should present _similar_ info.

**Comment by esirK** [comment]
> Cool, cool @esirK . So no need for Transnational in `28` itself? I'm assuming `28`, `28b` and `28c` are somewhat related and should present _similar_ info.

Right now we don't have it in `28`(might have been missed) but we can easily add it.

@kilemensi

**Pull request merged**